pkgs/development/python-modules/trezor/default.nix

```diff
@@ -44,6 +44,6 @@ buildPythonPackage rec {
     description = "Python library for communicating with TREZOR Bitcoin Hardware Wallet";
     homepage = "https://github.com/trezor/trezor-firmware/tree/master/python";
     license = lib.licenses.gpl3;
-    maintainers = with lib.maintainers; [ np prusnak mmahut "1000101" ];
+    maintainers = with lib.maintainers; [ np prusnak mmahut ] ++ [ lib.maintainers."1000101" ];
   };
```

Comment:
Perhaps use `meta = with lib;`.

Reply:
Also you can use `lib.maintainers.foo`  inside the first list – `with` statement just extends scope with unbound names.

Reply:
Also you can use `lib.maintainers.foo`  inside the first list – `with` statement just extends scope with unbound names.

Reply:
Very good point, one moment